### ISSUE TITLE
New Station : Gare_Couzon - little country station

### ISSUE THREAD

**Issue by Miksimus1** [Libre-TrainSim/Libre-TrainSim#342]
Hi !

I made a little country station.
![GareCouzon01](https://user-images.githubusercontent.com/98762775/154551661-88511930-eea1-4886-a0ad-ad7a625e228d.png)

I don't know ik it's ok like that, so I put here obj, mtl and blender files, if you want to modify it.

[Gare_Couzon.zip](https://github.com/Libre-TrainSim/Libre-TrainSim/files/8091445/Gare_Couzon.zip)

**Comment by DasCapschen** [comment]
First of all, thank you very much! That looks lovely!

What license are you willing to put this under?
I think last time I checked, @Jean28518 wanted only CC0 assets, but I think realistically we should allow any CC license, except non-derivative. 

I think @HaSa1002 could check the files for possible optimizations for Godot? :)

**Comment by Miksimus1** [comment]
MMmmm.... I hesitate...
In France the CC-BY and CC0 is the same thing because there are a law about "le droit moral de l’auteur" (moral copyright) : obligation for the user is to mention the author of the work.

If you think it's possible, i would put the CC-BY license.

And it's ok to check the file for Godot optimizations :-)

**Comment by HaSa1002** [comment]
Yes, I will check the file next time I work on LTS (There is a remote chance this is gonna be Saturday but realistically in two weeks).
I agree, we should totally accept CC BY and CC BY SA, too. Let's not get into the mess all the other creative commons licenses create.

This is a good issue to also mention that we should fix our credits to include our art and software licenses as well. We need to credit Godot and the software it uses and the addons we use for example. (And yes I am responsible for most of the addons 😅)

**Comment by Jean28518** [comment]
Great building! 
I just built some other stations some days ago, which can't compete with this one ;)

CC-BY or CC-BY-SA  are completely okay.

**Comment by Miksimus1** [comment]
Thanks ! :-D
I take advantage of this post to say that in the credits, the name "Miks" is enough for me ;-)
